src/spec/dsl.cr

```diff
@@ -230,7 +230,7 @@ module Spec
     start_time = Time.monotonic
 
     at_exit do
-      run_filters
+      root_context.run_filters(@@pattern, @@line, @@locations, @@split_filter, @@focus)
```

Comment:
I'd leave this line within the local method `run_filters`.

Reply:
You mean keep the local method that just does this one-liner?  I can do that.  I had figured the extra method wasn't worth it considering there are other calls to root_context (i.e. `root_context.run`) just below it and having them together in visual context made the code clearer to me.